tests/ui/GroupChatNameTests.tsx

```diff
@@ -0,0 +1,420 @@
+/* eslint-disable testing-library/no-node-access */
+import type * as NativeNavigation from '@react-navigation/native';
+import {act, fireEvent, render, screen, waitFor} from '@testing-library/react-native';
+import React from 'react';
+import {Linking} from 'react-native';
+import Onyx from 'react-native-onyx';
+import type Animated from 'react-native-reanimated';
+import * as Localize from '@libs/Localize';
+import * as AppActions from '@userActions/App';
+import * as User from '@userActions/User';
+import App from '@src/App';
+import CONST from '@src/CONST';
+import ONYXKEYS from '@src/ONYXKEYS';
+import appSetup from '@src/setup';
+import type {Participant} from '@src/types/onyx/Report';
+import PusherHelper from '../utils/PusherHelper';
+import * as TestHelper from '../utils/TestHelper';
+import waitForBatchedUpdates from '../utils/waitForBatchedUpdates';
+import waitForBatchedUpdatesWithAct from '../utils/waitForBatchedUpdatesWithAct';
+
+// We need a large timeout here as we are lazy loading React Navigation screens and this test is running against the entire mounted App
+jest.setTimeout(50000);
+
+jest.mock('../../src/components/ConfirmedRoute.tsx');
+
+// Needed for: https://stackoverflow.com/questions/76903168/mocking-libraries-in-jest
+jest.mock('react-native/Libraries/LogBox/LogBox', () => ({
+    /* eslint-disable-next-line @typescript-eslint/naming-convention */
+    __esModule: true,
+    default: {
+        ignoreLogs: jest.fn(),
+        ignoreAllLogs: jest.fn(),
+    },
+}));
+
+jest.mock('react-native-reanimated', () => ({
+    ...jest.requireActual<typeof Animated>('react-native-reanimated/mock'),
+    createAnimatedPropAdapter: jest.fn,
+    useReducedMotion: jest.fn,
+}));
```

Comment:
https://github.com/Expensify/App/blob/957b5ec8cd0803b1e7f878aabf6ddff7eeaabab5/tests/README.md#L48-L50

Can any of these be moved into `__mocks__` or no?

Reply:
What have you tried?

Reply:
Ok that's fine, but can we please add some explanation about why?